pom.xml

```diff
@@ -21,10 +21,10 @@ under the License.
     <parent>
         <groupId>org.apache.netbeans</groupId>
         <artifactId>netbeans-parent</artifactId>
-        <version>2</version>
+        <version>3</version>
     </parent>
     <groupId>org.apache.netbeans.native</groupId>
-    <version>12.6-SNAPSHOT</version>
+    <version>15.0-SNAPSHOT</version>
```

Comment:
If launchers are released separately, they should probably be versioned separately and differently to the IDE.

I'd go with `13.0-SNAPSHOT` or `2022.0-SNAPSHOT` - possibly the latter.

Reply:
using 2022-11 to have an idea of time and keep incrementing